.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
+}
```

Comment:
Please remove this file (and add `.vscode/` to the `.gitignore` file so similar files don't get added in the future).